eng/ci/templates/install-dotnet.yml

```diff
@@ -0,0 +1,19 @@
+steps:
+
+- task: UseDotNet@2 # Needed by some of our test resources
+  displayName: Install .NET 3.1
+  inputs:
+    packageType: sdk
+    version: 3.1.x
+
+- task: UseDotNet@2 # Needed by our projects and CI steps
+  displayName: Install .NET 6
+  inputs:
+    packageType: sdk
+    version: 6.x
```

Comment:
Do we need to install 8.x too at any point?

Reply:
The next task will do that - `Install .NET SDK from global.json`